roles/foreman_provisioning_infrastructure/tasks/compute_profiles.yml

```
@@ -1,22 +1,21 @@
 ---
-- name: 'find compute profile'
-  shell: >
-    {{ foreman_provisioning_hammer }} compute-profile info --name "libvirt-profile"
-  register: foreman_provisioning_compute_profile
-  ignore_errors: True
-
-- name: 'create compute profile'
-  shell: >
-    {{ foreman_provisioning_hammer }} compute-profile create
-    --name "libvirt-profile"
-  when: "'Error' in foreman_provisioning_compute_profile.stderr"
-
 - name: 'create compute attributes'
-  shell: >
-    {{ foreman_provisioning_hammer }} compute-profile values create
-    --compute-profile libvirt-profile
-    --compute-resource libvirt
-    --volume pool_name=provision,capacity=15G,format_type=qcow2
-    --interface type=network,network=provision,model=virtio
-    --compute-attributes cpus=1,memory=2400000000,start=1
-  when: "'Error' in foreman_provisioning_compute_profile.stderr"
+  theforeman.foreman.compute_profile:
+    name: libvirt-profile
+    compute_resource: libvirt
+    compute_attributes:
+      - compute_resource: libvirt
+        vm_attrs:
+          cpus: 1
+          memory: 2400000000
+          start: 1
+          nics_attributes:
+            0:
+              type: network
+              network: provision
+              model: virtio
```

@evgeni this is taken from the examples in https://theforeman.github.io/foreman-ansible-modules/develop/plugins/compute_profile_module.html#ansible-collections-theforeman-foreman-compute-profile-module but the description says it's a list, so should it be
```suggestion
            - type: network
              network: provision
              model: virtio
```